### ISSUE TITLE
Inductive Miner on DFG Bug

### ISSUE THREAD

**Issue by QuLars** [process-intelligence-solutions/pm4py#376]
Applying the inductive miner on the following DFG results in the error
```
TypeError: project() takes 3 positional arguments but 4 were given
```

```python
from pm4py import discover_petri_net_inductive
import pm4py.objects.dfg.obj as dfg_object
from collections import Counter

dfg = {
  ('a', 'b'): 1,
  ('b', 'a'): 1
}

start_activities = {'a': 1}
end_activities = {'a': 1}

dfg = dfg_object.DFG(Counter(dfg), Counter(start_activities), Counter(end_activities))
petri_net = discover_petri_net_inductive(dfg)
```

 (PM4Py version 2.4.1)

**Comment by fit-alessandro-berti** [comment]
Dear @QuLars 

Thanks for signaling. We identified the problem and will release the fix in the next release

**Comment by fit-alessandro-berti** [comment]
Dear @QuLars 

We resolved the aforementioned issue in pm4py 2.5.0
